aws/resource_aws_route53_resolver_endpoint.go

```diff
@@ -62,7 +63,7 @@ func resourceAwsRoute53ResolverEndpoint() *schema.Resource {
 						},
 					},
 				},
-				Set: route53ResolverHashIPAddress,
+				Set: route53ResolverEndpointHashIpAddress,
```

Comment:
Renamed `route53ResolverHashIPAddress` to `route53ResolverEndpointHashIpAddress` as it applies only to `aws_route53_resolver_endpoint`.